### DISCUSSION TITLE
How does this addon ensure @types/ember is compatible with the app's Ember version?

### DISCUSSION THREAD

**Discussion by GCheung55** [typed-ember/ember-cli-typescript#1456] [Q&A] [answered]
### Which package(s) does this enhancement pertain to?
  - [X] @types/ember
  - [ ] @types/ember__string
  - [ ] @types/ember__polyfills
  - [ ] @types/ember__object
  - [ ] @types/ember__utils
  - [ ] @types/ember__array
  - [ ] @types/ember__engine
  - [ ] @types/ember__debug
  - [ ] @types/ember__runloop
  - [ ] @types/ember__error
  - [ ] @types/ember__controller
  - [ ] @types/ember__component
  - [ ] @types/ember__routing
  - [ ] @types/ember__application
  - [ ] @types/ember__test
  - [ ] @types/ember__test-helpers
  - [ ] @types/ember__service
  - [ ] @types/ember-data
  - [ ] @types/rsvp
  - [ ] Other
  - [ ] I don't know

### Please write a user story for this feature

As a developer, I want to ensure that `@types/ember` is compatible with my application's version of Ember.js to prevent running into incompatible issues.

**Comment by chriskrycho** [comment, marked as answer]
Hi @GCheung55, so sorry we never answered this. I’ve moved it over to our new Discussions setup.

Unfortunately, the answer *today* (as when you asked it last year) is: you can’t! 😢 However, fixing this is one *huge* part of the reason we’re working to provide types natively with Ember as part of the story around [official TS support from the framework][rfc]: that will mean it stops being something you have to worry about.

[rfc]: https://github.com/emberjs/rfcs/pull/724
